libraries/helpers.rb

```diff
@@ -0,0 +1,19 @@
+module OSLMatrix
+  module Cookbook
+    module Helpers
+      # Generate a secret key. Mainly used when wanting a registration key
+      def osl_matrix_genkey(len = 64)
+        require 'securerandom'
+        SecureRandom.base64(len)
+      end
+
+      # Get the name of the synapse docker container name, given the name of the synapse resource which creates it
+      def osl_synapse_docker_name(strSynapseResource)
```

Comment:
I prefer names like this please:
```suggestion
      def osl_synapse_docker_name(synapse_resource)
```